spring-integration-core/src/main/java/org/springframework/integration/gateway/DefaultMethodInvokingMethodInterceptor.java

```diff
@@ -49,8 +49,7 @@
 
 	private final MethodHandleLookup methodHandleLookup = MethodHandleLookup.getMethodHandleLookup();
 
-	private final Map<Method, MethodHandle> methodHandleCache =
-			new ConcurrentReferenceHashMap<>(10, ReferenceType.WEAK);
+	private final Map<Method, MethodHandle> methodHandleCache = new ConcurrentReferenceHashMap<>(10, ReferenceType.WEAK);
```

Comment:
Why? It's now 121 chars.

Reply:
Sometimes I miss an assignment second line during review and I thought `121` is not too far from `120` 😄.

But rule is rule: reverted.